packages/console/src/pages/Connectors/components/ConnectorName.tsx

```diff
@@ -0,0 +1,42 @@
+import { ConnectorDTO } from '@logto/schemas';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+
+import ImagePlaceholder from '@/components/ImagePlaceholder';
+import ItemPreview from '@/components/ItemPreview';
+
+import * as styles from './ConnectorName.module.scss';
+
+type Props = {
+  connector?: ConnectorDTO;
+  titlePlaceholder?: string;
+};
+
+const ConnectorName = ({ connector, titlePlaceholder = '' }: Props) => {
+  const {
+    i18n: { language },
+  } = useTranslation();
+
+  if (!connector) {
+    return <ItemPreview title={titlePlaceholder} icon={<ImagePlaceholder />} />;
+  }
+
+  return (
+    <Link to={`/connectors/${connector.id}`} className={styles.link}>
```

Comment:
Maybe we can use `useNavigate` like this pr: https://github.com/logto-io/logto/pull/341/files#diff-3bf2e6d260a50bf321c44ec6ba69e775f03ff09a4e9fad3ae69274f479c67e5c